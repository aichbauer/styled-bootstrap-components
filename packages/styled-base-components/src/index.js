import { BootstrapBaseCss } from './components';

export { BootstrapBaseCss };
