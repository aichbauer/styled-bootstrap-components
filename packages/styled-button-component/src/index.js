import {
  Button,
  LinkButton,
} from './components/Button';
import { ButtonGroup } from './components/ButtonGroup';
import { ButtonToolbar } from './components/ButtonToolbar';

export {
  Button,
  LinkButton,
  ButtonGroup,
  ButtonToolbar,
};
