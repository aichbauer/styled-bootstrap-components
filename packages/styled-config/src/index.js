import { theme } from './theme';
import {
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
} from './utils';

export {
  theme,
  // utils
  border,
  borderRadius,
  boxShadow,
  colors,
  fontFamily,
  fontSize,
  fontWeight,
  height,
  margin,
  padding,
  screenSize,
  width,
};
