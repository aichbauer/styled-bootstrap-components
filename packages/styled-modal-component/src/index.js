import { Modal } from './components/Modal';
import { ModalWrapper } from './components/ModalWrapper';
import { ModalBody } from './components/ModalBody';
import { ModalContent } from './components/ModalContent';
import { ModalDialog } from './components/ModalDialog';
import { ModalFooter } from './components/ModalFooter';
import { ModalHeader } from './components/ModalHeader';
import { ModalTitle } from './components/ModalTitle';
import { ModalBackdrop } from './components/ModalBackdrop';

export {
  Modal,
  ModalWrapper,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  ModalBackdrop,
};
