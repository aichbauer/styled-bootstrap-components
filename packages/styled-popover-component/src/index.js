import { PopoverWithoutTransition, Popover } from './components/Popover';
import { PopoverArrow } from './components/PopoverArrow';
import { PopoverBody } from './components/PopoverBody';
import { PopoverHeader } from './components/PopoverHeader';

export {
  PopoverWithoutTransition,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
};
