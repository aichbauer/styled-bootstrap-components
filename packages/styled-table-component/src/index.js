import { Table } from './components/Table';
import { Tr } from './components/Tr';

export {
  Table,
  Tr,
};
