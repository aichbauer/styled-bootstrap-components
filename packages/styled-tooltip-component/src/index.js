import { TooltipWithoutTransition, Tooltip } from './components/Tooltip';
import { TooltipArrow } from './components/TooltipArrow';
import { TooltipInner } from './components/TooltipInner';

export {
  TooltipWithoutTransition,
  Tooltip,
  TooltipArrow,
  TooltipInner,
};
