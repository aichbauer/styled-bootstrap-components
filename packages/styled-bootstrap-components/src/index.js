import { Alert } from 'styled-alert-component';
import { Badge } from 'styled-badge-component';
import {
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  Button as BaseButton,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Nav as Navigation,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  Table as BaseTable,
  Textarea,
  Th,
  Ul,
} from 'styled-base-components';
import {
  Breadcrumb,
  BreadcrumbItem,
} from 'styled-breadcrumb-component';
import {
  Button,
  ButtonGroup,
  ButtonToolbar,
} from 'styled-button-component';
import {
  Card,
  CardBody,
  CardFooter,
  CardHeader,
  CardImageHeader,
  CardText,
  CardTitle,
} from 'styled-card-component';
import { screenSize } from 'styled-config';
import { Container } from 'styled-container-component';
import {
  Dropdown,
  DropdownItem,
  DropdownMenu,
  DropdownDivider,
} from 'styled-dropdown-component';
import {
  FormCheck,
  FormCheckInput,
  FormControl,
  FormControlFile,
  FormControlPlainText,
  FormGroup,
  FormInline,
  FormText,
  InputGroup,
  InputGroupAppend,
  InputGroupPrepend,
  InputGroupText,
} from 'styled-form-component';
import {
  Column,
  Row,
} from 'styled-grid-system-component';
import { Jumbotron } from 'styled-jumbotron-component';
import {
  ListGroup,
  ListGroupItem,
} from 'styled-listgroup-component';
import {
  Modal,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
} from 'styled-modal-component';
import {
  Nav,
  NavLink,
} from 'styled-nav-component';
import {
  Navbar,
  NavbarLink,
} from 'styled-navbar-component';
import {
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
} from 'styled-popover-component';
import {
  Table,
  Tr,
} from 'styled-table-component';
import {
  Tooltip,
  TooltipArrow,
  TooltipInner,
} from 'styled-tooltip-component';

export {
  Alert,
  Badge,
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  BaseButton,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Navigation,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  BaseTable,
  Textarea,
  Th,
  Ul,
  Breadcrumb,
  BreadcrumbItem,
  Button,
  ButtonGroup,
  ButtonToolbar,
  Card,
  CardBody,
  CardFooter,
  CardHeader,
  CardImageHeader,
  CardText,
  CardTitle,
  screenSize,
  Container,
  Dropdown,
  DropdownItem,
  DropdownMenu,
  DropdownDivider,
  FormCheck,
  FormCheckInput,
  FormControl,
  FormControlFile,
  FormControlPlainText,
  FormGroup,
  FormInline,
  FormText,
  InputGroup,
  InputGroupAppend,
  InputGroupPrepend,
  InputGroupText,
  Column,
  Row,
  Jumbotron,
  ListGroup,
  ListGroupItem,
  Modal,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  Nav,
  NavLink,
  Navbar,
  NavbarLink,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
  Table,
  Tr,
  Tooltip,
  TooltipArrow,
  TooltipInner,
};
