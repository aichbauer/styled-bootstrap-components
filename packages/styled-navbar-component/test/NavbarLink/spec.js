import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { NavbarLink } from '../../src';

test('Styles matches NavbarLink brand', () => {
  const tree = renderer.create(<NavbarLink brand />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink light active', () => {
  const tree = renderer.create(<NavbarLink light active />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink light disabled', () => {
  const tree = renderer.create(<NavbarLink light disabled />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink light disabled', () => {
  const tree = renderer.create(<NavbarLink light />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink dark active', () => {
  const tree = renderer.create(<NavbarLink dark active />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink dark disabled', () => {
  const tree = renderer.create(<NavbarLink dark disabled />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Styles matches NavbarLink dark disabled', () => {
  const tree = renderer.create(<NavbarLink dark />).toJSON();
  expect(tree).toMatchSnapshot();
});

