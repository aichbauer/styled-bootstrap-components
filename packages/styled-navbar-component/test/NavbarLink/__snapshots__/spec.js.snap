// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches NavbarLink brand 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #007bff;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(0,0,0,0.5);
  display: inline-block;
  padding-top: 0.3125rem;
  padding-bottom: 0.3125rem;
  font-size: 1.25rem;
  line-height: inherit;
  white-space: nowrap;
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #0056b3;
}

.c0:hover,
.c0:focus {
  color: rgba(0,0,0,0.7);
}

<a
  className="c0"
/>
`;

exports[`Styles matches NavbarLink dark active 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #007bff;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(255,255,255,0.9);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #0056b3;
}

.c0:hover,
.c0:focus {
  color: rgba(255,255,255,0.9);
}

<a
  className="c0"
/>
`;

exports[`Styles matches NavbarLink dark disabled 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #6c757d;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(255,255,255,0.25);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #6c757d;
}

.c0:hover,
.c0:focus {
  color: rgba(255,255,255,0.25);
}

<a
  className="c0"
  disabled={true}
/>
`;

exports[`Styles matches NavbarLink dark disabled 2`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #007bff;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(255,255,255,0.5);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #0056b3;
}

.c0:hover,
.c0:focus {
  color: rgba(255,255,255,0.75);
}

<a
  className="c0"
/>
`;

exports[`Styles matches NavbarLink light active 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #007bff;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(0,0,0,0.9);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #0056b3;
}

.c0:hover,
.c0:focus {
  color: rgba(0,0,0,0.9);
}

<a
  className="c0"
/>
`;

exports[`Styles matches NavbarLink light disabled 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #6c757d;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(0,0,0,0.3);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #6c757d;
}

.c0:hover,
.c0:focus {
  color: rgba(0,0,0,0.3);
}

<a
  className="c0"
  disabled={true}
/>
`;

exports[`Styles matches NavbarLink light disabled 2`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  color: #007bff;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: rgba(0,0,0,0.5);
}

.c0:hover,
.c0:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover,
.c0:focus {
  color: #0056b3;
}

.c0:hover,
.c0:focus {
  color: rgba(0,0,0,0.7);
}

<a
  className="c0"
/>
`;
